stacker/providers/aws/default.py

```diff
@@ -207,18 +208,46 @@ def destroy_stack(self, stack, **kwargs):
         return True
 
     def create_stack(self, fqn, template_url, parameters, tags, **kwargs):
-        logger.debug("Stack %s not found, creating.", fqn)
-        logger.debug("Using parameters: %s", parameters)
-        logger.debug("Using tags: %s", tags)
-        retry_on_throttling(
-            self.cloudformation.create_stack,
-            kwargs=dict(StackName=fqn,
-                        TemplateURL=template_url,
-                        Parameters=parameters,
-                        Tags=tags,
-                        Capabilities=["CAPABILITY_NAMED_IAM"]),
-        )
-        return True
+        try:
+            logger.debug("Stack %s not found, creating.", fqn)
+            logger.debug("Using parameters: %s", parameters)
+            logger.debug("Using tags: %s", tags)
+            logger.debug("Using template_url: %s", template_url)
+            retry_on_throttling(
+                self.cloudformation.create_stack,
+                kwargs=dict(StackName=fqn,
+                            TemplateURL=template_url,
+                            Parameters=parameters,
+                            Tags=tags,
+                            Capabilities=["CAPABILITY_NAMED_IAM"]),
+            )
+            return True
+        except botocore.exceptions.ClientError as e:
+            if e.response['Error']['Message'].startswith('TemplateURL'):
+                logger.warn("DEPRECATION WARNING: Falling back to legacy "
```

Comment:
This is complex enough that we might want a document that describes the change.  I believe this is it:

- There was a "bug" early on in stacker that created the s3 bucket in `us-east-1` no matter what you specified as your `--region`
- There was an issue that this wasn't the expected behavior, so we fixed the behavior (so that the bucket is created in the correct region specified).  As well we added the `stacker_bucket_region` option to give people more control.
- If someone has been using stacker for a while, their bucket is in `us-east-1`, and they are launching stacks in other regions, this will now break due to the new logic.  The way to fix this is to either migrate their bucket somehow (probably not a real strategy) or to set `stacker_bucket_region` to us-east-1.

Then we can have a brief message here that points to the URL of the doc that explains it a bit better.